pandas/tests/series/test_reindex_like.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+
+import numpy as np
+
+from pandas import Series
+import pandas._testing as tm
+
+
+def test_reindex_like(datetime_series):
+    other = datetime_series[::2]
+    tm.assert_series_equal(
+        datetime_series.reindex(other.index), datetime_series.reindex_like(other)
+    )
+
+    # GH#7179
+    day1 = datetime(2013, 3, 5)
+    day2 = datetime(2013, 5, 5)
+    day3 = datetime(2014, 3, 5)
+
+    series1 = Series([5, None, None], [day1, day2, day3])
+    series2 = Series([None, None], [day1, day3])
+
+    result = series1.reindex_like(series2, method="pad")
+    expected = Series([5, np.nan], index=[day1, day3])
+    tm.assert_series_equal(result, expected)
+
+
+def test_reindex_like_nearest():
+    s = Series(np.arange(10, dtype="int64"))
+    target = [0.1, 0.9, 1.5, 2.0]
+    result = s.reindex(target, method="nearest")
```

Comment:
hmm, some of these shouldn't be moved here (e.g. the onces that are just nearest reindex)

Reply:
good catch, updated